src/com/lftechnology/java/training/alina/jdbc/domain/EmployeeRole.java

```diff
@@ -0,0 +1,15 @@
+package com.lftechnology.java.training.alina.jdbc.domain;
+
+/**
+ * EmployeeRole Class defines roles of employee
+ * 
+ * @author Alina Shakya <alinashakya@lftechnology.com>
+ */
+public enum EmployeeRole {
+    ADMIN("admin"), USER("user");
+    public String role;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make role a static final constant or non-public and provide accessors if needed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)